examples/storybook/src/stories/VoiceSearch.stories.ts

```diff
@@ -0,0 +1,116 @@
+        <ais-voice-search></ais-voice-search>
+      `,
+    }),
+  }))
+  .add('without status', () => ({
+    component: wrapWithHits({
+      template: `
+        <ais-voice-search>
+          <ng-template #status></ng-template>
+        </ais-voice-search>
+      `,
+    }),
+  }))
+  .add('with a ais-search-box', () => ({
+    component: wrapWithHits({
+      template: `
+        <ais-voice-search></ais-voice-search>
+        <ais-search-box></ais-search-box>
+      `,
+    }),
+  }))
+  .add('with a custom button text', () => ({
+    component: wrapWithHits({
+      template: `
+        <div class="custom-button">
+          <ais-voice-search>
+            <ng-template #button let-isListening="isListening">
+              {{isListening ? '⏹' : '🎙'}}
+            </ng-template>
+          </ais-voice-search>
+        </div>
+      `,
+    }),
+  }))
+  .add('with full status', () => ({
+    component: wrapWithHits({
+      template: `
+        <ais-voice-search>
+          <ng-template #status let-status="status"
+                               let-errorCode="errorCode"
+                               let-isListening="isListening"
+                               let-transcript="transcript"
+                               let-isSpeechFinal="isSpeechFinal"
+                               let-isBrowserSupported="isBrowserSupported">
+            <p>status: {{status}}</p>
+            <p>errorCode: {{errorCode}}</p>
+            <p>isListening: {{isListening}}</p>
+            <p>transcript: {{transcript}}</p>
+            <p>isSpeechFinal: {{isSpeechFinal}}</p>
+            <p>isBrowserSupported: {{isBrowserSupported}}</p>
+          </ng-template>
+        </ais-voice-search>
+      `,
+    }),
+  }))
+  .add('with custom button title', () => ({
+    component: wrapWithHits({
+      template: `
+        <ais-voice-search
+          buttonTitle="Voice Search"
+          disabledButtonTitle="Disabled. Chrome Only."
+        >
+        </ais-voice-search>
+      `,
+    }),
+  }))
+  .add('search as you speak', () => ({
+    component: wrapWithHits({
+      template: `
+        <ais-voice-search
+          [searchAsYouSpeak]="true"
+        >
+          <ng-template #status let-status="status"
+                               let-errorCode="errorCode"
+                               let-isListening="isListening"
+                               let-transcript="transcript"
+                               let-isSpeechFinal="isSpeechFinal"
+                               let-isBrowserSupported="isBrowserSupported">
+            <p>status: {{status}}</p>
+            <p>errorCode: {{errorCode}}</p>
+            <p>isListening: {{isListening}}</p>
+            <p>transcript: {{transcript}}</p>
+            <p>isSpeechFinal: {{isSpeechFinal}}</p>
+            <p>isBrowserSupported: {{isBrowserSupported}}</p>
+          </ng-template>
+        </ais-voice-search>
+      `,
+    }),
+  }))
+  .add('example of dynamic UI working with SearchBox', () => ({
```

Comment:
I noticed a small glitch on this one as the voice search button overlaps the clear button of the search box.
It is not the case on the [vanilla demo](https://instantsearchjs.netlify.com/stories/?path=/story/voicesearch--example-of-dynamic-ui-working-with-searchbox). Can we avoid it in Angular IS too?

Reply:
Yes, I missed this. I removed the reset button on the vanilla demo. Now that I don't see that option in angular-IS, I will hide the button with css.

Reply:
https://github.com/algolia/angular-instantsearch/pull/530/commits/7c9655ea4e273245fcc37a91ae61724dcdd3f387